ksp_plugin/plugin.cpp

```diff
@@ -502,13 +502,16 @@ void Plugin::IncrementPartIntrinsicForce(PartId const part_id,
 void Plugin::IncrementPartIntrinsicForceWithPosition(
     PartId const part_id,
     Vector<Force, World> const& force,
-    Position<World> const& position) {
+    Position<World> const& position,
+    Position<World> const& part_position) {
   CHECK(!initializing_);
   not_null<Vessel*> const vessel = FindOrDie(part_id_to_vessel_, part_id);
   CHECK(is_loaded(vessel));
-  // TODO(egg): Do something with position.
   vessel->part(part_id)->increment_intrinsic_force(
       renderer_->WorldToBarycentric(PlanetariumRotation())(force));
+  vessel->part(part_id)->increment_intrinsic_torque(
```

Comment:
I would prefer to have an overload of `increment_intrinsic_force` that would, roughly, do what lines 510-514 do.  Physics logic should be avoided in the plugin.